model-optimizer/extensions/front/tf/ObjectDetectionAPI.py

```diff
@@ -1185,14 +1198,162 @@ def generate_sub_graph(self, graph: Graph, match: SubgraphMatch):
+                                                          softmax_conf_node)
+
+        # prepare input with box logits
+        boxes_logit = backward_bfs_for_operation(match.single_input_node(0)[0], ['Add'])[0]
+        reshape_box_logits = create_op_node_with_second_input(
+            graph, Reshape, int64_array([0, -1]), dict(name=boxes_logit.soft_get('name', boxes_logit.id) + '/Flatten'),
+            boxes_logit)
+
+        yxyx_anchors = match.single_input_node(2)[0]
+
+        variance_height = pipeline_config.get_param('frcnn_variance_height')
+        variance_width = pipeline_config.get_param('frcnn_variance_width')
+        variance_x = pipeline_config.get_param('frcnn_variance_x')
+        variance_y = pipeline_config.get_param('frcnn_variance_y')
+
+        # get the input image height and width to divide the anchors values by it
+        initial_input_node_name = 'input_tensor' if 'input_tensor' in graph.nodes else 'image_tensor'
+        if initial_input_node_name not in graph.nodes():
+            raise Error('Input node "{}" of the graph is not found. Do not run the Model Optimizer with '
+                        '"--input" command line parameter.'.format(initial_input_node_name))
+        parameter_node = Node(graph, initial_input_node_name)
+
+        input_shape = Shape(graph, {'name': parameter_node.name}).create_node([parameter_node])
+        input_image_hw = node_to_get_shape_value_of_indices(input_shape, [1, 2])  # NHWC layout
+        hwhw = create_op_with_const_inputs(graph, Tile, {1: int64_array([2])}, {'name': 'image_hwhw'}, input_image_hw)
+
+        hwhw_float = Cast(graph,
+                          {'dst_type': data_type_str_to_np(graph.graph['cmd_params'].data_type)}).create_node([hwhw])
+        scaled_anchors = Div(graph, {'name': 'scaled_anchors'}).create_node([yxyx_anchors, hwhw_float])
+
+        flattened_anchors = create_op_with_const_inputs(graph, Reshape, {1: int64_array([1, 1, -1])},
+                                                        {'name': 'flattened_anchors'}, scaled_anchors)
+        cropped_anchors = AttributedClamp(graph, {'min': 0.0, 'max': 1.0, 'name': 'clamped_xyxy',
+                                                  'nchw_layout': True}).create_node([flattened_anchors])
+        mark_as_correct_data_layout(flattened_anchors)
+
+        # create tensor of shape [4] with variance values which then are tiled by the number of boxes which is obtained
+        # from the 'yxyx_anchors' node
+        variances = Const(graph, {'value': np.float32([1.0 / variance_x, 1.0 / variance_y, 1.0 / variance_width,
+                                                       1.0 / variance_height])}).create_node()
+
+        anchors_shape = Shape(graph, {'name': 'anchors_shape'}).create_node([yxyx_anchors])
+        anchors_count = node_to_get_shape_value_of_indices(anchors_shape, [0])
+        tiled_anchors = Tile(graph, {'name': 'tiled_anchors'}).create_node([variances, anchors_count])
```

Comment:
it is not a tensor of anchors. Can we rather to rename it to `tiled_variances`?

Reply:
Correct. Done.